phases/phase-1/challenges/referred/Alive29

```diff
@@ -0,0 +1 @@
+AndreyVelde
```

Comment:
Currently @AndreyVelde has not yet referred you. Do you want to wait for him to refer you or would you like to drop this referral?

Reply:
Fixed in #53